cypress/pageobjects/rancher.po.ts

```diff
@@ -0,0 +1,497 @@
+import type { CypressChainable } from '@/utils/po.types';
+import { Constants } from '../constants/constants';
+import SettingsPagePo from "@/pageobjects/settings.po";
+import LabeledInputPo from '@/utils/components/labeled-input.po';
+import LabeledSelectPo from '@/utils/components/labeled-select.po';
+import RadioButtonPo from '@/utils/components/radio-button.po'
+import { List } from 'cypress/types/lodash';
+
+
+const constants = new Constants();
+const settings = new SettingsPagePo();
+var registrationURL;
+
+interface ValueInterface {
+    namespace?: string,
+    name: string,
+    description?: string,
+    url?: string,
+    size?: string,
+    path?: string,
+    labels?: any,
+    harvester: string,
+    cloud_credential: string,
+    rke2_cluster_name: string,
+}
+
+
+
+export class rancherPage {
+    
+    private login_page_usernameInput = '#username';
+    private login_page_passwordInput = '#password > .labeled-input > input';
+    private login_page_loginButton = '#submit > span';
+    private main_page_title = '.title';
+    private dashboardURL = 'dashboard/home';
+
+    private boostrap_page_welcome = '.info-box > :nth-child(1)';
+    private boostrap_page_boostrapPWInput = 'input';
+    private boostrap_page_boostrapPWSubmit = '#submit > span';
+    private boostrap_page_radioSelectPW = ':nth-child(2) > .radio-container';
+    private boostrap_page_newPWInput = ':nth-child(5) > .password > .labeled-input > input';
+    private boostrap_page_newPWRepeat = '[style=""] > .labeled-input > input';
+    // private boostrap_page_checkAgreeEULA = '#checkbox-eula > .checkbox-container > .checkbox-custom';
+    private boostrap_page_checkAgreeEULA = '[data-testid="setup-agreement"] > .checkbox-container > .checkbox-custom';
+    private boostrap_page_confirmLogin = '.btn > span';
+    private home_page_mainMenu = '.menu';
+    private home_page_virtualManagement = ':nth-child(7) > .option > div';
+
+    private virtual_page_importButton = '.btn';
```

Comment:
<img width="690" alt="image" src="https://user-images.githubusercontent.com/24985926/201635588-056f5bc1-e028-45d9-a8d3-dd8edcd721e0.png">

This will find multiple elements.

Reply:
Update the virtual_page_importButton to specific locator
  ```
  virtual_page_importButton = '.btn.role-primary';
  ```